Branch-SDK/src/io/branch/referral/BranchRemoteInterface.java

```diff
@@ -270,23 +272,26 @@ public void sendLog(String log) {
 		}
 	}
 	
-	private String convertJSONtoString(JSONObject json) {
+	private String convertJSONtoString(JSONObject json) throws JSONException {
```

Comment:
We are still catching JSONException down there, do we still need to throw it here?